docs/book/src/reference/writing-tests.md

```diff
@@ -3,7 +3,7 @@
 Testing Kubernetes controller is a big subject, and the boilerplate testing
 files generated for you by kubebuilder are fairly minimal.
 
-[Writing and Running Integration Tests](/reference/testing/integration.md) documents steps to consider when writing integration steps for your controllers, and available options for configuring your test control plane using [`envtest`](https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/envtest).
+[Writing and Running Integration Tests](testing/integration.md) documents steps to consider when writing integration steps for your controllers, and available options for configuring your test control plane using [`envtest`](https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/envtest).
```

Comment:
> testing/integration.md

After taking a closer look at `docs/testing/integration.md` and `docs/testing/e2e.md`, I just realized that they are for kubebuilder v1 and are out-of-date.
You will need to do the following:
- Change the link to `/reference/testing/envtest.md`. (Please ensure it works in netlify preview)
- Please add a note on top of `docs/testing/integration.md` and `docs/testing/e2e.md`. e.g. `**This document is for kubebuilder v1 only**`
- Revert other changes in `SUMMARY.md` and `integration.md`.

Sorry for the confusion.
I will add a small doc for e2e testing afterwards.

Reply:
IIRC, we intentionally left this out of the book since they were v1 articles.  We need to add v2 equivalents, but they aren't there yet.

Reply:
Thank you both, I understood. It wasn't clear to me.
Going to fix.